### ISSUE TITLE
Invalid JSON output

### ISSUE THREAD

**Issue by khamaileon** [Cap-go/CLI#218]
```bash
npx @capgo/cli bundle zip --json
```

This command provides an incorrect json for each new version of capgo/cli. It may be useful locally, but not in a build process.

```
│
▲  🚨 You are using @capgo/cli@3.14.60 it's not the latest version.
│  Please use @capgo/cli@3.14.61" or @capgo/cli@latest to keep up to date with the latest features and bug fixes.
{"bundle":"1.16.0","filename":"app.example.1.16.0.zip","checksum":"803073f2"}
```

**Comment by fxsalazar** [comment]
#199 was made to fix this

**Comment by khamaileon** [comment]
With each new version of capgo/cli, our build pipeline is broken. I see @fxsalazar has proposed a fix. What's the reason for this PR #199 to be closed?

**Comment by riderx** [comment]
Hello,
Sorry i missed this issue,
I checked #199, and I found it a bit complex, where I could just disable the check in the CI.
I just committed a fix for it: https://github.com/Cap-go/CLI/commit/540363461a6d2410e8e3130d70a0b0def2186c9f

**Comment by riderx** [comment]
available in : [4.0.12](https://github.com/Cap-go/CLI/commit/46fa71a939f88cce5e05a515b47b46f8b8631e83)

**Comment by khamaileon** [comment]
@riderx it doesn't work anymore on the last version 😓
